backend/app/api/uploads.py

```
@@ -0,0 +1,100 @@
+    b"\x4d\x3c\xb2\xa1",  # Little-endian Nanosecond PCAP
+    b"\x0a\x0d\x0d\x0a",  # PCAPNG Section Header Block
+}
+
+# Maximum upload size: 100 MB
+MAX_UPLOAD_SIZE = 100 * 1024 * 1024
+
+
+class SizeLimitExceeded(Exception):
+    """Exception raised when the uploaded file exceeds the allowed size."""
+
+    pass
```

Ruff (0.15.15)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)